### DISCUSSION TITLE
Is possible to use F5 and Ctrl+F5 instead of F9 and Ctrl+F9 to debug and run without debug?

### DISCUSSION THREAD

**Discussion by pravinsable** [mark-wiemer/ahkpp#176] [General]
Is possible to use F5 and Ctrl+F5 instead of F9 and Ctrl+F9 to debug and run without debug?

**Comment by mark-wiemer** [comment]
Yes, you are able to customize any keyboard shortcuts in VS Code, see [VS Code keyboard shortcuts](https://code.visualstudio.com/docs/getstarted/keybindings).

I don't want to change the defaults as F5 is already taken and again, users can modify keyboard shortcuts to fit their needs very easily.

**Reply by pravinsable** [reply]
I was able get this keys right by changing existing Ctrl+F5 expression 
debuggersAvailable && debugState != 'initializing' && editorLangId != 'ahk'

now Ctrl+F5 runs AutoHotKey  script without holding up editor. 

Can we add "AHK++ : Run AHK Script" command to the editor toolbar on top right? Currently it only has Debug button. There is option to hide it but no option to show Run button. This is standard in SciTeAutohotkey editor.

**Comment by kyklish** [comment]
I don't remember exactly: when you press `F5` VS Code says there are no debugger for ahk and show drop-down list of available debuggers for ahk where you choose AHK++.debugger by default for ahk. After that you can use default hotkeys for debug and maybe run.
